backend/main_test.go

```diff
@@ -476,3 +465,80 @@ func TestImproveBadMethods(t *testing.T) {
 	postresp := makeHTTPRequest(postreq, handler, t)
 	checkResponseStatus(http.StatusMethodNotAllowed, postresp, t)
 }
+
+type failingAuthStub struct{}
```

Comment:
Nit: we should probably be explicit that it's *authentication* (rather than authorization) that's failing here. `failAuthenticationStub` maybe?

Reply:
👍🏻